### PR TITLE
XS-3264: Update next_q1_16 with master

### PR DESCRIPTION
**Pull request by chrislococo-wf**
### Changes:

- Updated next_q1_16 with master.
- Deleted files are replaced with up to date files with no version in the name.

### Pull request process

- [x] Full description of changes provided above.
- [x] Reviewed by another person. (+1)
- [x] If this is a code change, reviewed by a second person. (+1)
- [x] QA'd to the specifications listed [here](https://github.com/DataQualityCommittee/dqc_us_rules#quality-assurance-qa-of-a-pull-request). Documentation only changes do not require the formal code quality assurance process. (+10)

### CODE REVIEW

File: dqc_us_rules/dqc_us_0005.py [hunk 1]
```diff
@@ -11,7 +11,9 @@
     re.compile(r'^http://xbrl\.((sec\.gov)|(us))/dei/\d{4}-\d{2}-\d{2}')
 )
 _CODE_NAME = 'DQC.US.0005'
-_RULE_VERSION = '1.0'
+_RULE_VERSION = '1.1.0'
+
+_REPORT_TYPE_EXCLUSIONS = ['S-1', 'S-11']
```
Comment (jasonleinbach-wf) [inline dqc_us_rules/dqc_us_0005.py]: Looks like a botched merge here - _REPORT_TYPE_EXCLUSIONS is defined twice.

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by mindeerickson-wf** [comment]
+1

**Comment by mindeerickson-wf** [comment]
Testing:
* Verified Travis ran successfully:  https://travis-ci.org/DataQualityCommittee/dqc_us_rules/builds/145675614
* Ran a few of the DQC 15 test docs through arelle and compared the output I received to that listed in the variations.xml file to verify that rule is working as expected

QA +1

**Comment by andrewperkins-wf** [comment]
+1 on last changes.  Just changes to configuration files, so QA +10 once the CI build finishes.

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by andrewperkins-wf** [comment]
Merging

**Pull request merged**